degirum_tools/zone_count.py

```diff
@@ -382,7 +382,7 @@ def annotate(self, result, image: np.ndarray) -> np.ndarray:
                 text,
                 tuple(x + line_width for x in self._polygons[zi][0]),
                 font_color=text_color,
-                bg_color=line_color,
+                bg_color=line_color[::-1],

```

Comment:
Well, this change is questionable:
1. model.overlay_color is RGB
2. so are line_color and text_color
3. put_text() accepts colors in RGB

Why do you need to revert color here?